controller/applicationsController.js

```diff
@@ -0,0 +1,67 @@
+const Admin = require("../models/admins");
+const Organization = require("../models/organizations");
+const Application = require("../models/applications");
+const { getAppToken } = require("../utils/auth/tokenGenerator");
+const { ObjectId } = require("mongoose").Types;
+
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+
+const createApplication = async (req, res, next) => {
+  try {
+    //validation should be done via middleware
+    //ownerId in body also needs to be validated
+
+    const { name, organizationId, createdBy } = req.body;
+
+    if (!ObjectId.isValid(organizationId) || !ObjectId.isValid(createdBy)) {
```

Comment:
Please split these conditional tests to specifically report if the `organizationId` is in invalid or if `createdBy` is invalid, to properly inform the user where they need to make corrections